docs/ui-extensions-sdk-frontend.md

```diff
@@ -22,29 +22,18 @@ communicate with the Contentful Management App.
 You will need to include the `contentful-extension-sdk` library in your HTML5 app like
 so:
 ~~~html
-<script src="https://contentful.github.io/ui-extensions-sdk/cf-extension-api.js"></script>
+<script src="https://unpkg.com/contentful-ui-extensions-sdk@2"></script>
```

Comment:
how about having `@latest` here instead of `@major` this way we encourage to use the latest version and we don't have to update it each time